README.md

```diff
@@ -21,9 +21,13 @@
 - [🙏 Acknowledgements](#acknowledgements)
 - [📝 License](#license)
 
-# 📖 **BLOG APP** <a name="about-project"></a>
+# 📖 **BUDGET BUDDY** <a name="about-project"></a>
 
-> A **BUDGET APP** is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+> A **BUDGET BUDDY** is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+> To know more about the app, watch this [Video Presentation](https://drive.google.com/file/d/15xbmuU3o9ElexqBmzX12OQOQS2WAlvBB/view?usp=sharing) !
+
+> [Live Demo Link](https://budget-application-xjpp.onrender.com/) of the app !
```

Comment:
Nice job deploying your project as required👍🏼. currently, when I try to create an account, I consistently get this error preventing the action from being successful😉.
<img width="418" alt="image" src="https://github.com/anita00001/budget-app/assets/64643995/07d49f69-316f-4556-9845-05d1d82bc0f8">

Please do well to address it as not only is it a requirement that our projects are deployed but the live version should also work properly😉 and apart from that, the live version of your project will serve as a good way to show others your app and such obstruction in the application can lead to a poor user experience😉.

Reply:
Thank you so much for your suggestion. I have updated the live demo link, and it should work now!